client/components/ProductDetails.tsx

```diff
@@ -64,35 +74,72 @@ const ProductDetails = () => {
       </div>
 
       <div className="flex justify-between items-center mb-4 p-2 border-y border-primary-light">
-        <div className="flex flex-col gap-2">
-          <p className="text-gray-700">{product.price} USD por hora</p>
-          <p className="text-gray-500">Marca: {product.brand}</p>
-          <div className="flex flex-row gap-2 justify-start">
-            <Check color="#0eba69" />
-            <span className="text-[var(--color-active)] font-bold">
-              {product.status}
-            </span>
+        <div className="flex flex-col gap-4">
+          {" "}
+          {/* Increase gap for consistent spacing */}
+          {/* Price */}
+          <div className="flex items-center gap-2">
+            {" "}
+            {/* Flex row with consistent gap */}
+            <DollarSignIcon className="text-gray-500" />
+            <p className="text-gray-700">
+              <strong>{product.price} USD</strong> por hora
+            </p>
+          </div>
+          {/* Brand */}
+          <div className="flex items-center gap-2">
+            {" "}
+            {/* Flex row with consistent gap */}
+            <CrownIcon className="text-gray-500" />
+            <p className="text-gray-500">Marca: {product.brand}</p>
+          </div>
+          {/* Status */}
+          <div className="flex items-center gap-2">
+            {" "}
+            {/* Flex row with consistent gap */}
+            {product.status ? (
+              <>
+                <Check className="text-green-500" /> {/* Adjust icon size/color if needed */}
+                <span className="text-[var(--color-active)] font-bold">Disponible</span>
+              </>
+            ) : (
+              <>
+                <XIcon className="text-red-600" /> {/* Replace this with your preferred "x" icon */}
+                <span className="text-red-600 font-bold">No disponible</span>
+              </>
+            )}
           </div>
         </div>
-        <button className="bg-yellow-500 text-white font-semibold py-2 px-4 rounded-md hover:bg-yellow-600">
-          Reservar
-        </button>
       </div>
+
       {/* Descripción */}
       <div className="mb-4">
         <h2 className="text-xl font-semibold">Descripción</h2>
         <p className="text-gray-700">{product.description}</p>
-        <a href="#" className="text-blue-600" onClick={() => openModal(0)}>
+        {/* <a href="#" className="text-blue-600" onClick={() => openModal(0)}>
           Ver más
-        </a>
+        </a> */}
+      </div>
+
+      <div className="flex justify-center items-center mb-4 p-2">
+        <div className="flex flex-col gap-2">
+          <a href="#" className="text-yellow-600 mr-12 underline" onClick={() => openModal(0)}>
+            Ver más
+          </a>
+        </div>
+        <button
+          className={`py-4 px-24 rounded-full ml-12 font-normal ${
+            product.status
+              ? "bg-yellow-500 text-black hover:bg-yellow-600"
```

Comment:
There are colors in variables. Replace bg-yellow for "bg-secondary". The same applies to the rest of colors like gray-300